docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-tensorflow.rst

```diff
@@ -0,0 +1,111 @@
+defined in the previous step. This method returns a so-called compression controller and a wrapped model that can be used the
+same way as the original model. Optimization methods are applied at this step, so that the model
+undergoes a set of corresponding transformations and contains additional operations required for optimization. In case of QAT, the compression controller object is used for model export and, optionally, in distributed training as demonstrated below.
+
+.. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+   :language: python
+   :fragment: [wrap_model]
+
+
+4. Fine-tune the Model
+####################################
+
+This step assumes applying fine-tuning to the model the same way it is done for the baseline model. For QAT, it is required to train the model for a few epochs with a small learning rate, for example, 10e-5. In principle,
+you can skip this step, meaning that the post-training optimization will be applied to the model.
+
+.. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+   :language: python
+   :fragment: [tune_model]
+
+
+5. Multi-GPU Distributed Training
+####################################
+
+In the case of distributed multi-GPU training (not DataParallel), call ``compression_ctrl.distributed()`` before fine-tuning. This informs optimization methods to make adjustments to function in the distributed mode.
+
+.. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+   :language: python
+   :fragment: [distributed]
+
+
+.. note::
+   The precision of weights transitions to INT8 only after converting the model to OpenVINO Intermediate Representation.
+   You can expect a reduction in model footprint only for that format.
+
+
+These steps outline the basics of applying the QAT method from the NNCF. However, in some cases, it is required to save/load model
+checkpoints during training. Since NNCF wraps the original model with its own object, it provides an API for these needs.
+
+6. (Optional) Save Checkpoint
+####################################
+
+To save a model checkpoint, use the following API:
+
+.. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+   :language: python
+   :fragment: [save_checkpoint]
+
+
+7. (Optional) Restore from Checkpoint
+################################################
+
+To restore the model from checkpoint, use the following API:
+
+.. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+   :language: python
+   :fragment: [load_checkpoint]
+
+
+For more details on saving/loading checkpoints in the NNCF, see the corresponding `NNCF documentation <https://github.com/openvinotoolkit/nncf/blob/develop/docs/Usage.md#saving-and-loading-compressed-models>`__.
+
+Deploying the Quantized Model
+###############################
+
+The model can be converted into the OpenVINO Intermediate Representation (IR) if needed, compiled, and run with OpenVINO without any additional steps.
```

Comment:
@tsavina, seems like `,` before the `and` should not be there: the model can be converted, compiled and run with OV...
Please correct me if I'm wrong